src/components/CreateTodoModal.js

```diff
@@ -12,6 +12,7 @@ export default function CreateTodoModal({
   addTodo,
 }) {
   const newTodo = useRef();
+  const newTodoDescription =useRef();
```

Comment:
new reference `newTodoDescription` which is never used.

Reply:
I did it - see in the next merge request